python/graphscope/framework/graph.py

```diff
@@ -432,6 +432,7 @@ def add_column(self, results, selector):
         op = dag_utils.add_column(self, results, selector)
         graph = Graph(self._session, op)
         graph._base_graph = self
+        graph._ensure_loaded()
```

Comment:
Why `add_column` requires such force loading however `project` (as well as `project_to_simple`) doesn't need such hacking?

Reply:
It's @acezen thinks that the error should be raised in the method, not later. and `_project_to_simple` does have some checking as it has more information in the python side.

Reply:
I see.

Leave a note/comment here would be better.